Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -1862,6 +1921,11 @@ - (WKPageRef)_pageForTesting
     return _page.get();
 }
 
+- (RetainPtr<STWebpageController>)_screenTimeWebpageController
```

Comment:
I think typically we just return the raw reference like `STWebpageController *` and then the implementation will be `return _screenTimeWebpageController.get()`